core/src/tangram.h

```diff
@@ -154,6 +156,7 @@ bool getDebugFlag(DebugFlags _flag);
 void toggleDebugFlag(DebugFlags _flag);
 
 void runOnMainLoop(std::function<void()> _task);
+void runAsyncTask(std::function<void()> _task);
```

Comment:
True, it's probably more `runOnMainLoop` that can be confusing if you don't know what is a tangram main loop.